packages/core/src/command.ts

```diff
@@ -62,33 +62,29 @@ class Command {
               cmdStr += ','
             }
 
-            // safely append the val - avoid blowing up when attempting to
-            // call .replace() if message is not a string for some reason
-            cmdStr += `${key}=${escape(`${val || ''}`)}`
+            cmdStr += `${key}=${escapeProperty(val)}`
```

Comment:
I was reading it and thinking that eventually, the function being exported, it is diable for users to run `issueCommand('some-command', {'key,1': 'value'}, 'message')` that would break the `::name key1=value1,key2=value2::message` formatting. would it be something worth taking into account?

Reply:
It is a limitation of the command format - [defined](https://github.com/actions/runner/blob/a9c58d7398d463b0dc5d7d1ca1a8f99d2005a808/src/Runner.Common/ActionCommand.cs) by the runner - that commas are not allowed within a key name.

Today this limitation does not present a problem. All commands (and keys) are first party - not extensible.

We can relax this constraint in the future needed. But not required for any current scenarios.